spamostack/keeper.py

```diff
@@ -119,6 +120,10 @@ def get(self, client_name, resource_name, param=None, func=None,
                     else:
                         probe = getattr(el, param)(*args, **kwargs)
 
+                    if (resource_name == "flavors"
```

Comment:
Sure, if you could just pass an id which will contain `spamo-` as prefix and then just check it in `spam_factory.py` delete method of flavor that would be a better solution :)